.doctrine-project.json

```diff
@@ -4,21 +4,39 @@
     "slug": "cache",
     "docsSlug": "doctrine-cache",
     "versions": [
+        {
+            "name": "2.1",
+            "branchName": "2.1.x",
+            "slug": "2.1",
+            "current": true
+        },
+        {
+            "name": "2.0",
+            "branchName": "2.0.x",
+            "slug": "2.0",
+            "maintained": false
+        },
+        {
+            "name": "1.12",
+            "branchName": "1.12.x",
+            "slug": "1.12",
+            "maintained": false
```

Comment:
I'd still flag the 1.12.x branch as maintained (for everything related to PSR-6) and merge bugfixes like #387 to it. Doing so shouldn't be much overhead since the PSR-6 wrappers are identical on 1.12.x and 2.1.x.

Reply:
Oh right, I got fooled by the "merged into master" in https://github.com/doctrine/cache/pull/360 . Makes sense to retarget.